katalog/curator-s3/curator-add-task.yml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: snapshot-configurator
+spec:
+  schedule: '0 0 * * *'
+  suspend: true
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: curator
+              image: quay.io/sighup/curator
+              envFrom:
+                - secretRef:
+                    name: curator-aws
+              command:
+                - python
+              args:
+                - /scripts/add_task.py
```

Comment:
Why is this script needed?

Reply:
if you read it, you'll understand it :)

Reply:
I don't. I read it, I see it posts some data (bucket, and credentials) to ES API. But I don't understand what is this for and why it should run before triggering the curator.

Reply:
is the way to configure the plugin s3 bucket of elasticsearch, it must be done through an HTTP POST

Reply:
ok, I don't get why we need to execute every time you run the curator, but I can live with it

Reply:
is an idempotent http post request, so it won't have any side effect :)